osmosis-1/osmosis.zone_assets.json

```diff
@@ -1566,6 +1566,13 @@
       "path": "transfer/channel-1411/umpwr",
       "osmosis_main": false,
       "osmosis_frontier": true
+    },
+    {
+      "chain_name": "juno",
+      "base_denom": "unexx",
+      "path": "transfer/channel-42/factory/juno1qly4zcmzr2gyxtze5yt9chv2srczwwunppxjfh/NEXX",
```

Comment:
Please register this token to the [Chain Registry](https://github.com/cosmos/chain-registry/blob/master/juno/assetlist.json)